### DISCUSSION TITLE
右侧能否实现一个滚动条

### DISCUSSION THREAD

**Discussion by flynnxin0730** [wanghongenpin/proxypin#207] [General]
右侧能否做一个滚动条，就像GitHub 手机app 右侧 的竖线滚动条

**Comment by flynnxin0730** [comment]
![image](https://github.com/wanghongenpin/network_proxy_flutter/assets/25835363/7f4a4d3d-7112-46a8-824f-563d69f40b21)

**Comment by wanghongenpin** [comment]
抓包列表吗

**Comment by wanghongenpin** [comment]
V1.1.2版本已经支持

https://github.com/wanghongenpin/network_proxy_flutter/releases/tag/V1.1.2

**Reply by flynnxin0730** [reply]
thanks
